src/plugins/intel_cpu/tests/functional/utils/cpu_test_utils.cpp

```diff
@@ -410,30 +410,6 @@ void CPUTestsBase::updateSelectedType(const std::string& primitiveType,
     selectedType += execType.get_type_name();
 }
 
-std::vector<CPUSpecificParams> filterCPUSpecificParams(const std::vector<CPUSpecificParams>& paramsVector) {
```

Comment:
Let's revert this change and fix the particular test cases which cause the issue

Reply:
This change is reverted (this method has been moved to `utils/x64/filter_cpu_info.cpp`
2 methods - `getCPUParamsBlocked4D` and `getCPUParamsBlocked5D` were implemented in `single_layer_tests/slice.cpp` to do x64-specific fintering